azure-kusto-data/azure/kusto/data/kcsb.py

```diff
@@ -128,12 +128,13 @@ def __init__(self, connection_string: str):
         for kvp_string in connection_string.split(";"):
             key, _, value = kvp_string.partition("=")
             keyword = self.ValidKeywords.parse(key)
+            value_stripped = value.strip()
             if keyword.is_str_type():
-                self[keyword] = value.strip()
-            if keyword.is_bool_type():
-                if value.strip() in ["True", "true"]:
+                self[keyword] = value_stripped.rstrip("/")
+            elif keyword.is_bool_type():
+                if value_stripped in ["True", "true"]:
```

Comment:
MMMM, this means that "True/" is a valid value here, which is not.
You only need the regular strip() here for spaces and such, not removing the trailing slash.